docs/netobserv_cli.adoc

```diff
@@ -0,0 +1,107 @@
+// Automatically generated by './scripts/generate-doc.sh'. Do not edit, or make the NETOBSERV team aware of the editions.
+:_mod-docs-content-type: REFERENCE
+[id="network-observability-cli-usage_{context}"]
+= Network Observability CLI usage
+
+You can use the The Network Observability CLI (`oc netobserv`) to pass command line arguments to capture flow data and packet data for further analysis, enable Network Observability Operator features or pass configuration options to the eBPF agent and flowlogs-pipeline.
+== Syntax 
+The basic syntax for `oc netobserv` commands: 
```

Comment:
I think you need a carriage return between 8 and 9, otherwise the rendering is wonky

Reply:
https://github.com/netobserv/network-observability-cli/pull/85/commits/38470dbfd5b30f5e94a68873d5f32f1e4c76e999